controllers/activemqartemis_controller.go

```diff
@@ -461,6 +461,19 @@ func UpdateCRStatus(cr *brokerv1beta1.ActiveMQArtemis, client rtclient.Client, n
 		return err
 	}
 
+	if len(cr.Status.ExternalConfigs) != len(current.Status.ExternalConfigs) {
+		return resources.UpdateStatus(client, cr)
+	}
+	if len(cr.Status.ExternalConfigs) >= 0 {
+		for _, cfg := range cr.Status.ExternalConfigs {
+			for _, curCfg := range current.Status.ExternalConfigs {
+				if curCfg.Name == cfg.Name && curCfg.ResourceVersion != cfg.ResourceVersion {
+					return resources.UpdateStatus(client, cr)
+				}
+			}
+		}
+	}
+
```

Comment:
@gtully could you move this change in a separate commit?

Reply:
yep, either that or verify the external config status in one of those tests. I omitted doing that. let me revisit in the next day or so. thanks for the feedback.

Reply:
I have added a test to validate the need for that fix. thanks!